ineffassign.go

```diff
@@ -35,7 +36,13 @@ func main() {
 		}
 		fmt.Fprintln(os.Stderr, "\nFlags:")
 		flag.PrintDefaults()
+		ineffassign.Analyzer.Flags.PrintDefaults()
 	}
+	// maddeningly, this is how analysis/singlechecker inits its flags
```

Comment:
singlechecker passes all analyzers to this code: https://cs.opensource.google/go/x/tools/+/refs/tags/v0.33.0:go/analysis/internal/analysisflags/flags.go;l=59-67